projects/packages/videopress/src/client/block-editor/blocks/video/edit.tsx

```diff
@@ -144,52 +144,8 @@ export default function VideoPressEdit( {
 	const token = useMediaToken( { guid, isPrivate, tokenType: 'playback' } );
 
 	// Store if the chapters file is auto-generated in a local state.
```

Comment:
I'm wondering if this comment should be along with the following line in the hooks `useAutoGeneratedChapter`, WDYT?
```
const [ isAutoGeneratedChapter, setIsAutoGeneratedChapter ] = useState( true );
```